### ISSUE TITLE
Feature: Streamlined UI

### ISSUE THREAD

**Issue by cloaked2222** [HW-PlayersPatch/Development#148]
Eventually would be nice to improve the UI. 

The Minimalist UI Mod by Lifth / Lif looks quite nice:
https://steamcommunity.com/sharedfiles/filedetails/?id=1466520028&searchtext=

Need something more usable for MP though. The above deletes the whole selection menu which is key for hw2 dock micro. We could add another tab to the options to hide/show different UI elements as the user desires.

**Comment by Novaras** [comment]
Did you hear from the author of minimalist UI mod?

**Comment by cloaked2222** [comment]
No, I tried steam/gbx forums. I doubt he games that often, next time he loads steam i'm sure he'll respond.
